aws/cmd/template.go

```diff
@@ -75,7 +75,7 @@ const (
 			for hasNextToken {
 				o, err := c.svc.{{.Service}}.{{.ServiceEntityFn}}WithContext(ctx, input)
 				if err != nil {
-					return nil, err
+					return nil, fmt.Errorf("%w: %v", errcode.ErrProviderAPI, err)
```

Comment:
I would not add this to the template but to the `Provider#Resources` as is the end caller, so we could have only 1 line and not this repeated on all the functions.

Also I would not always fail, but check, potentially on this case see if it matches `Unavailable Operation` to then wrap it so it'll be ignored, but let the rest still make the CLI fail.